### PR TITLE
Moving all resources to be top level

### PR DESCRIPTION
**Pull request by dblane-digicatapult**
This PR should move everything to being top level prior to migrating the repo to being .github

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Pull Request
 
 ## Checklist
-- [ ] Have you read {project_name}'s Code of Conduct? https://github.com/digicatapult/{project_module_name}/blob/main/CODE_OF_CONDUCT.md
+- [ ] Have you read Digital Catapult's [Code of Conduct](/../CODE_OF_CONDUCT.md)?
```
Comment (mattdean-digicatapult) [inline .github/PULL_REQUEST_TEMPLATE.md:4]: Do we know if relative links like this will work as a .github?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -37,19 +37,19 @@ We don't have any frequently asked questions yet.
 
 ### Reporting Bugs
 
-This section guides you through submitting a bug report for {project_module_name}. Following these guidelines helps maintainers and the community understand your report :pencil:, reproduce the behaviour :computer: :computer:, and find related reports :mag_right:.
+This section guides you through submitting a bug report for the project. Following these guidelines helps maintainers and the community understand your report :pencil:, reproduce the behaviour :computer: :computer:, and find related reports :mag_right:.
```
Comment (mattdean-digicatapult) [inline CONTRIBUTING.md:40]: It's fairly clear you've done a find replace to switch to `the project`. I think the english can be cleaned up a bit by simply remove it completely in many cases. For example here I think it sounds better just being `This section guides you through submitting a bug report`. Can you look through the changes you've made and see where such simplifications still make sense. Simpler is generally better

File: resources/.github/CODEOWNERS
Comment (mattdean-digicatapult) [inline resources/.github/CODEOWNERS:1]: Do we know if a `.github` repo will apply codeowners universally?

File: .github/ISSUE_TEMPLATE/bug_report.yml
Comment (mattdean-digicatapult) [inline .github/ISSUE_TEMPLATE/bug_report.yml:1]: This is such a better format!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (dblane-digicatapult) [reply .github/PULL_REQUEST_TEMPLATE.md:4]: No idea, need to create a second repo to test it, and need this one to be in place to test it.

File: resources/.github/CODEOWNERS
Comment (dblane-digicatapult) [reply resources/.github/CODEOWNERS:1]: From what I've found, it does not.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (dblane-digicatapult) [reply .github/PULL_REQUEST_TEMPLATE.md:4]: If it doesn't work, then I can just open another PR which links to this repo's versions.

**Review by mattdean-digicatapult — APPROVED** [review]

**Pull request merged**